onnx/helper.py

```diff
@@ -965,6 +965,7 @@ def make_optional(
         raise TypeError("The element type in the input optional is not supported.")
 
     assert value is not None
+    assert attribute is not None
```

Comment:
If that happens, I would add an error message to give the attribute type. The user can leverage that information. With no message, he has to print it himself to understand how to fix his issue.

Reply:
In this case there is not much to say because attribute is None.

Reply:
What I meant is why it is None. elem_type is not one of the supported cases. I think it would be better to tell that than saying nothing.

Reply:
I see, it's possible to remove the assertion here without causing mypy warnings.